src/Database/Schema/SqliteSchema.php

```diff
@@ -340,6 +339,14 @@ public function columnSql(TableSchema $schema, $name)
             }
         }
 
+        if ($data['type'] === TableSchema::TYPE_BINARY) {
+            if (isset($data['length'])) {
+                $out .= ' VARBINARY(' . (int)$data['length'] . ')';
+            } else {
+                $out .= ' BLOB';
+            }
```

Comment:
I don't know if these types are actually different in sqlite. They might be cosmetically different  but behave the same.